kong/init.lua

```diff
@@ -768,6 +730,10 @@ function Kong.access()
     ctx.workspace = old_ws
   end
 
+  if ctx.has_response then
+    kong.ctx.core.buffered_proxying = true
+  end
+
```

Comment:
Should we move this closer to usage? I mean there are a returns and response.exits below, and in that case we are not buffered proxying?